SignInCheckIn/MinistryPlatform.Translation/Repositories/ParticipantRepository.cs

```diff
@@ -84,17 +84,19 @@ public List<MpEventParticipantDto> GetChildParticipantsByEvent(string token, int
             return children;
         }
 
-        public MpNewParticipantDto CreateParticipantWithContact(string authenticationToken, MpNewParticipantDto mpNewParticipantDto)
+        public MpNewParticipantDto CreateParticipantWithContact(MpNewParticipantDto mpNewParticipantDto)
         {
-            var token = authenticationToken ?? _apiUserRepository.GetToken();
```

Comment:
I think we still want to use the authentication token if provided as I believe it is used for the audit logs.